server.test.js

```diff
@@ -25,20 +29,133 @@ describe('Games', () => {
   });
   // declare some global variables for use of testing
   // hint - these wont be constants because you'll need to override them.
+  let newGame = new Game({
```

Comment:
You might want to define this inside the `beforeEach` hook, simply because it's only used within the scope of that hook.